src/plugins/auto/auto_schedule.hpp

```diff
@@ -48,6 +48,8 @@ class AutoSchedule : public MultiSchedule {
 
 public:
     AutoLoadContext                           _loadContext[CONTEXTNUM];
+    // The flag to show if AUTO will call MULTI logic
+    bool                                      _AutoCallMulti = {false};
```

Comment:
probably better with private member

Reply:
In this case, it can not be accessed in src/plugins/auto/auto_executable_network.cpp L170 if private.